Project/Source/Components/Trail.cpp

```diff
@@ -0,0 +1,298 @@
+#include "Trail.h"
```

Comment:
Should be Trail Class now outside the Components folder? Since now it's not a component @JaimeBea

Reply:
idk, i leav it there couse its more easy to find it and its seems like that but i agree with all

Reply:
It would be better to leave it outside for now, just to stay consistent.